lib/config.go

```diff
@@ -545,8 +545,15 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 // `nil`.
 func (c *NetworkConfig) Validate(onExecution bool) error {
 	if onExecution {
-		if err := utils.IsDirectory(c.NetworkDir); err != nil {
-			return errors.Wrapf(err, "invalid network_dir: %s", err)
+		err := utils.IsDirectory(c.NetworkDir)
+		if err != nil {
+			if os.IsNotExist(err) {
+				if err = os.MkdirAll(c.NetworkDir, 0755); err != nil {
+					return errors.Wrapf(err, "invalid network_dir: %s", err)
```

Comment:
please change the error message to something like `"cannot create network_dir: %s", c.NetworkDir`

I see the issue was already existing, but since we are touching this part let's fix it.  There is no need to repeat `err` in the error message.  Having `c.NetworkDir` is more useful.

Reply:
Done. :)